### ISSUE TITLE
Padding needs to be removed at bottom of each section of display, and Layout needs to be fixed in a couple ways

### ISSUE THREAD

**Issue by ZacharyLesser** [upenndigitalscholarship/deep#186]
There is extra spacing at the bottom of Stationer Information, Title-Page Features, etc.

**Comment by ZacharyLesser** [comment]
Here's a view of the layout of each record that we'd like. IGNORE the color-coding. Currently it is functioning correctly -- fields that should always appear and that should only appear when there is data in them are working correctly.

The problem is that there are some blank lines that should not be there, and there is padding at the bottom of each section. 

The only time a blank line should appear is when it is listed here as "[blank line]". Otherwise, if a field does NOT appear, because there is no data in it, it should not be replaced by a blank line but should instead be closed up. 

So, for instance, if there is nothing in the "Variants" field, there should only be ONE blank line between "Total Editions" and whatever comes next, or if nothing comes after it (because no data in any of the fields below "Total Editions") then the horizontal line should come immediately after "Total Editions"

In other words, blank lines inserted between fields that only appear when data is present should themselves only appear when needed to separate these fields.


[Layout.of.Record.pdf](https://github.com/upenndigitalscholarship/deep/files/15252899/Layout.of.Record.pdf)

**Comment by ZacharyLesser** [comment]
Here's how it looks around the horizontal line on the old site -- this is what we would like, so the lines are pretty tight up against that horizontal line:

![ORpDxdkVDYnCtphD](https://github.com/upenndigitalscholarship/deep/assets/40445411/4e3d23f4-7c67-4e7b-a660-33ca68088239)
